src/component/UI/Header/Header.scss

```diff
@@ -0,0 +1,107 @@
+$cardo: 'Cardo', serif;
+$dropdown-content: #f1f1f1;
+$dropdown-content-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+$primary-color: #606E86;
+$secondary-color : #ffff;
+$reem-kufi: 'Reem Kufi', sans-serif;
```

Comment:
Hi @el-Joft,
Please make use of the global scss variables for consistency. For instance `$reem-kufi` has already been defined. Thanks